data/strings/sound.txt

```diff
@@ -10,6 +10,7 @@
     de = Halten Sie sich rechts.
     el = Μείνετε δεξιά.
     es = Manténgase a la derecha.
+    es-MX = Manténgase a la derecha.
```

Comment:
Will our engine work if duplicate sub-language strings are removed if they are equal to the main language (e.g. es == es-MX, pt == pt-BR, etc.)? Other strings work fine. I would prefer to remove duplicate sub-language translations to keep it simpler.

Reply:
We'll find out! Certain TTS-specific words like "onto" do need to be specified but I've removed all other duplication.